CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
-## 0.8.2-dev6
+## 0.8.2
 
 ### Enhancements
 
+* Pass parameter `pdf_infer_table_structure` in partition_image to support image table extration
```

Comment:
Small nit: partition_image -> `partition_image` (code ticks)

Reply:
this changelog is outdated, if you refresh you could see the latest one

Reply:
Yeah I noticed that after I posted and couldn't figure out how to uncomment 😂